web/impact/impact/views/algolia_api_key_view.py

```diff
@@ -19,23 +21,38 @@ class AlgoliaApiKeyView(APIView):
     actions = ["GET"]
 
     def get(self, request, format=None):
+        index_prefix = settings.ALGOLIA_INDEX_PREFIX
         client = algoliasearch.Client(
             settings.ALGOLIA_APPLICATION_ID,
             settings.ALGOLIA_API_KEY)
         params = {
             'hitsPerPage': 24,
-            'filters': '',
             'validUntil': int(time.time()) + 3600,
             'userToken': request.user.id
         }
         search_key = settings.ALGOLIA_SEARCH_ONLY_API_KEY
         if request.user.is_staff:
             search_key = settings.ALGOLIA_STAFF_SEARCH_ONLY_API_KEY
-
+        else:
+            active_roles = UserRole.FINALIST_USER_ROLES
+            active_roles.append(UserRole.MENTOR)
+            facet_filters = []
+            for grant in request.user.programrolegrant_set.filter(
+                    program_role__program__program_status__in=CURRENT_STATUSES,
```

Comment:
This means that Alumni loose access to seeing mentors that they once were able to see, which seems unfortunate to me.

Reply:
alumni in this case should end up seeing all confirmed mentors since it should fall back to using is_confirmed_mentor: true as a filter.